Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -500,6 +500,11 @@ public typealias StartPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
     static func setDefaultInstance(_ purchases: Purchases) {
         self.purchases.modify { currentInstance in
             if currentInstance != nil {
+                #if DEBUG
+                if ProcessInfo.isRunningUnitTests {
+                    preconditionFailure(Strings.configure.purchase_instance_already_set.description)
+                }
+                #endif
```

Comment:
does that mean that any apps that have tests using our SDK will also crash here? If so we should find a way to limit this to just _our_ testing, a customer should be able to do whatever they want in their tests

Reply:
Oh that's a fair point. But considering this is undesired usage of the SDK, and crashing during tests has no user impact, isn't that a good thing?
Running the SDK concurrently can lead to the concurrent post receipt operations issue.